config/locales/en.yml

```diff
@@ -466,6 +468,7 @@ en:
       aspect_selector: "Aspect selector button"
       aspect_selector_ex1: "This button is how you select who will be able to read your post. It is set to “All aspects” by default: that is, everyone you have added to one of your aspects will be able to read the message, but no one else will be. In the simple message you just sent, because you wanted it visible just to your followers and the default is “All aspects,” there was no need to change any settings before sharing your message."
       aspect_selector_ex2: "With this button you can select any individual aspect to share with, or any combination of your aspects, by clicking on aspects in the list to select or unselect them. In this way you have complete control over who gets to read your messages, as we discussed in %{part3_link}. If, on the other hand, you want to announce something to the whole world, select “Public,” and there will be no restrictions on who can read your message."
+      aspect_selector_ex3: "Note: you can change the default visibility of your posts to whatever you like in your user settings – see %{part7_link}."
```

Comment:
Naming this `part7_link` isn't a good idea too, or it will break in the future when we re-order parts, see other comment for more information. Since this is a new link (it isn't broken now, but can be a problem in the future), maybe just name it `part_link` without number?